docs/organizations/managing-people.md

```diff
@@ -0,0 +1,66 @@
+# Managing people
+
+Members of an organization can see the details of the repositories in that organization and add new repositories to be analyzed by Codacy. Besides this, **Codacy only analyzes commits to repositories from contributors who are members of the corresponding organization on Codacy**.
+
+!!! important
+    Make sure that your team members join your organization on Codacy so that Codacy analyzes their commits in the repositories of that organization.
+
+!!! note
+    The organizations on Codacy that contain your personal repositories can only have the owner of the repositories as their single member.
```

Comment:
"Not true". You can add that email as author and it will be a billed seat. I would just remove this to avoid confusion. 

Personal orgs and normal orgs don't have any differences regarding seats(members) the "difference" that exists is that another member can not join your personal organization. But you can add emails do analyze freely and **it will update your people list accordingly.** 
I don't think this "difference" is worth mention. Specially because we will now not show any major visual difference between a "user" and "author" it would just confuse the user. It is just: People under this list are the ones being billed.

Reply:
If you want to maintain the "Bitbucket Cloud" use case I understand and you can. But you could mention the real use case that is available only for BB which is "the personal organization showing up to them" not sure if a documentation about the People page is the right one.

Reply:
Thanks for the clarification @inesgomas, and your suggestions do make sense: it's better to just mention the "positive" exception that users can join someone else's personal organizations on Bitbucket. :+1:

Reply:
Thinking a bit more about this, I'm considering not even mentioning the Bitbucket "exception" at all, since on this Git provider there's not really the concept of "personal repository". Instead, all repositories must belong to a Workspace:

![image](https://user-images.githubusercontent.com/60105800/106480964-d7eccf80-64a3-11eb-96eb-64bafeef2bc6.png)

These Workspaces map to Codacy organizations, but once again there's no real mapping to the concept of "personal organization":

![image](https://user-images.githubusercontent.com/60105800/106481264-28fcc380-64a4-11eb-97e7-6aea15f813f8.png)

I think that until we have more information that tells us otherwise, it's simpler for the users if we omit this detail. :+1:

Reply:
That is also true 😄 Maybe not mentioning is the best